template/clean.sls

```diff
@@ -0,0 +1,7 @@
+# -*- coding: utf-8 -*-
+# vim: ft=sls
+
+include:
+  - .service.clean
+  - .config.clean
+  - .pkg.clean
```

Comment:
For consistency with `service` and `config` & `future` modules I suggest `pkg` be changed to `package`.

Reply:
OK, that can be done.

Reply:
Done.